samples/BikeSharingApp/README.md

```diff
@@ -1,4 +1,103 @@
+
+1. **Deploy the sample application to Kubernetes.** We'll use Helm to run this sample application, but other tooling could be used to run your entire application in a namespace within a cluster. The Helm commands are targeting the namespace named `master` you created earlier, and can take several minutes to complete.
+    ```bash
+    cd charts/
+    helm init --wait
+    helm install -n bikesharing . --dep-up --namespace master --atomic
+    ```
+    Note: **If you are using an RBAC-enabled cluster**, be sure to configure [a service account for Tiller](https://helm.sh/docs/using_helm/#role-based-access-control). Otherwise, `helm` commands will fail.
+
+1. **Open your browser to the app's website.** Run the `azds list-uris` command to see the public endpoints in the running app. Navigate to the `bikesharingweb` service - in the below example, the public URL for the `bikesharingweb` service is http://master.bikesharingweb.fedcab0987.eus.azds.io/. Select **Aurelia Briggs (customer)** as the user, then select a bike to rent.
+    ```bash
+    azds list-uris
+
+    Uri                                                   Status
+    --------------------------------------------------    ---------
+    http://master.bikesharingweb.fedcab0987.eus.azds.io/  Available
+    http://master.gateway.fedcab0987.eus.azds.io/         Available
+    ```
+
+## Next Steps
+Now that you have the BikeSharing app deployed in AKS, try these walkthroughs try these walkthroughs to learn how Dev Spaces can enhance the Kubernetes development experience:
+
+1. **[Use your public endpoint in the cloud to privately debug backend code that’s running on your local dev machine.](https://aka.ms/devspaces/connect)** This minimizes what you need to set up on your dev machine – the only thing you need to run on your machine is the microservice you’re working on and your preferred dev tools, no need to set up mocks or simulators. You don’t even need Kubernetes YAML or Docker configuration to do this, and you won’t affect the currently deployed app or anyone who’s using the AKS cluster.
+
+1. **[Combine GitHub Actions with Dev Spaces in a pull request review.](https://aka.ms/devspaces/pr-flow)** You can use GitHub Actions to automatically deploy to a new sandbox whenever a pull request is opened so that your team can review a live version of the app that includes your pull request changes – all before that code is merged into your main branch! As a bonus, team members such as product managers and designers can become part of the review process during early stages of development.
+
+1. **[Debug and iterate code directly in AKS.](https://docs.microsoft.com/azure/dev-spaces/quickstart-netcore)** This is similar to the first scenario, except this mode enables a *higher fidelity development and testing experience* by helping you build and run a container image directly in AKS. Dev Spaces can help you generate Docker and Kubernetes assets.
```

Comment:
I suggest making this 2 and GitHub 3.  Also, how about this for content: This is similar to the first scenario, except this mode enables a *higher fidelity development and testing experience* by running your code as a container directly in AKS.  Dev Spaces will help you generate the necessary assets to build and debug in the AKS cluster.

Reply:
Done